pom.xml

```diff
@@ -106,8 +106,19 @@
     <commons-compress.version>1.18</commons-compress.version>
     <commons-fileupload.version>1.3.3</commons-fileupload.version>
 
-    <!-- spring-security version -->
-    <spring-security.version>4.1.5.RELEASE</spring-security.version>
+    <!-- spring version -->
+    <spring-version>4.3.19.RELEASE</spring-version>
```

Comment:
Please comply with naming conventions. This applies to all version properties. 
Check https://github.com/pentaho/pentaho-coding-standards/wiki/1.3.-Managing-Dependencies#version-specification
Namely:
- Use the `.version` suffix for all properties that pertain version numbers.
- Do not use `-` separator for the version suffix.

In this case the property would be called something like `<springframework.version>`.

Reply:
Or use `<spring.PROJECTNAME.version>` ... so `<spring.framework.version>`,  `<spring.boot.version>`, `<spring.security.version>`, etc

Reply:
@graimundo can you please confirm that the correct format will be the following? I want to confirm it because this will require some find and replaces in other PR's, just to make sure that is according all conventions. 

   <!-- spring version -->
    <spring.version>4.3.19.RELEASE</spring.version>
    <spring-ldap.version>2.3.2.RELEASE</spring-ldap.version>
    <spring-binding.version>2.4.8.RELEASE</spring-binding.version>
    <spring-se-jcr.version>0.9</spring-se-jcr.version>
    <spring-mock.version>2.0.8</spring-mock.version>
    <spring-osgi.version>1.2.1</spring-osgi.version>
    <spring-security.version>4.2.9.RELEASE</spring-security.version>
    <spring-security-core-tests.version>2.0.5.RELEASE</spring-security-core-tests.version>
    <spring-retry.version>1.2.2.RELEASE</spring-retry.version>
    <spring-boot.version>1.5.16.RELEASE</spring-boot.version>
    <spring-security-kerberos.version>1.0.1.RELEASE</spring-security-kerberos.version>
    <karaf-springframework-override.version>4.1.9.RELEASE_1</karaf-springframework-override.version>

Reply:
@graimundo can you please confirm that the correct format will be the following? Because @pentaho-lmartins said that you were discussing this topic and only the version should be separated by "." . I want to confirm it because this will require some find and replaces in other PR's, just to make sure that is according all conventions.

   <!-- spring version -->
    <spring.version>4.3.19.RELEASE</spring.version>
    <spring-ldap.version>2.3.2.RELEASE</spring-ldap.version>
    <spring-binding.version>2.4.8.RELEASE</spring-binding.version>
    <spring-se-jcr.version>0.9</spring-se-jcr.version>
    <spring-mock.version>2.0.8</spring-mock.version>
    <spring-osgi.version>1.2.1</spring-osgi.version>
    <spring-security.version>4.2.9.RELEASE</spring-security.version>
    <spring-security-core-tests.version>2.0.5.RELEASE</spring-security-core-tests.version>
    <spring-retry.version>1.2.2.RELEASE</spring-retry.version>
    <spring-boot.version>1.5.16.RELEASE</spring-boot.version>
    <spring-security-kerberos.version>1.0.1.RELEASE</spring-security-kerberos.version>
    <karaf-springframework-override.version>4.1.9.RELEASE_1</karaf-springframework-override.version>